src/main/java/kr/codesqaud/cafe/controller/UserController.java

```diff
@@ -1,58 +1,57 @@
 package kr.codesqaud.cafe.controller;
 
-import kr.codesqaud.cafe.dto.UserDTO;
-import kr.codesqaud.cafe.repository.MemoryUserRepository;
-import kr.codesqaud.cafe.repository.UserRepository;
+import kr.codesqaud.cafe.controller.dto.request.JoinRequest;
+import kr.codesqaud.cafe.controller.dto.request.ProfileEditRequest;
 import kr.codesqaud.cafe.service.UserService;
-import kr.codesqaud.cafe.user.User;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.List;
-
 @Controller
-@RequestMapping("/users")
 public class UserController {
-    private final MemoryUserRepository memoryUserRepository;
+    private final UserService userService;
 
-    @Autowired
-    public UserController(MemoryUserRepository memoryUserRepository) {
-        this.memoryUserRepository = memoryUserRepository;
+    public UserController(UserService userService) {
+        this.userService = userService;
     }
 
-    @GetMapping("/user/form")
-    public String join() {
+    @GetMapping("/create-user-form")
+    public String showJoinForm() {
         return "user/form";
     }
 
-    @GetMapping
-    public String listAllUsers(Model model){
-        List<User> users = memoryUserRepository.findAll();
-        model.addAttribute("users", users);
-        return "user/list";
+    @GetMapping("/login-user-form")
+    public String showLoginForm() {
+        return "user/login";
     }
 
-    @PostMapping("/user/create")
-    public String create(UserDTO userDTO) {
-        User user = new User(userDTO.getUserId(), userDTO.getPassword(),
-                userDTO.getName(), userDTO.getEmail());
-        memoryUserRepository.save(user);
+    @PostMapping("/join")//생성
```

Comment:
```suggestion
    @PostMapping("/join")
```

코드를 단순히 설명하는 주석은 굳이 안적어주셔도 이해하는데 큰 무리는 없습니다~